src/lib/ui/thermostat/ThermostatCard.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { useDevice } from '../../../hooks.js'
+import { ThermostatCard } from './ThermostatCard.js'
+
+const meta: Meta<typeof ThermostatCard> = {
+  title: 'Library/ThermostatCard',
+  tags: ['autodocs'],
+  component: ThermostatCard,
+}
+
+type Story = StoryObj<typeof ThermostatCard>
+
+export const Content: Story = {
+  render: () => {
+    const { device } = useDevice({
+      device_id: 'device5',
+    })
+
+    if (device == null) return <></>
+
```

Comment:
```suggestion
    if (device == null) return null
    if (!isThermostatDevice(device)) return null
```

Reply:
The story needs to return something through all paths, right? I did this because it would complain on `null`.